docs/users-and-roles-swagger.yaml

```diff
@@ -360,6 +381,25 @@ components:
             $ref: '#/components/schemas/AccessRight'
         viewingScope:
           $ref: '#/components/schemas/ViewingScope'
+    CreateUserRequest:
+      properties:
+        type:
+          type: string
+          required: true
+          enum:
+            - 'CreateUserRequest'
```

Comment:
You dont use this "type" in CreateUserREquest. Is it needed?

Reply:
It seems like we provide object types in given swagger schema for almost all objects, but value for those is set to string, which is in my opinion strange/not correct. So when frontenders will consume post vs put endpoints, they can differentiate between request objects, in this case CreateUserRequest vs User:


```
{
  "type": "CreateUserRequest",
  "nin": "string",
  "customerId": "string",
  "roles": [
    {
      "type": "string",
      "rolename": "string",
      "accessRight": [
        "APPROVE_DOI_REQUEST"
      ]
    }
  ],
  "viewingScope": {
    "includedUnits": [
      "string"
    ]
  }
} 
```

vs

```
{
  "type": "User",
  "username": "string",
  "givenName": "string",
  "familyName": "string",
  "institution": "string",
  "roles": [
    {
      "type": "string",
      "rolename": "string",
      "accessRight": [
        "APPROVE_DOI_REQUEST"
      ]
    }
  ],
  "accessRight": [
    "APPROVE_DOI_REQUEST"
  ],
  "viewingScope": {
    "includedUnits": [
      "string"
    ]
  }
}
```